<launch>
	<group ns="fmLib">
		<node pkg="can_socketcan" type="can_socketcan_node" name="can">
			<param name="device" value="can0" />
			<param name="publisher_topic" value="/fmCSP/can0_rx" />
			<param name="subscriber_topic" value="/fmCSP/can0_tx" />
		</node>
		<node pkg="serial_nmea" name="nmea_node" type="nmea_node" output="screen">
			<param name="str_to_msg_sub" value="/fmData/omnirota_rx" />   
			<param name="str_to_msg_pub" value="/fmData/nmea_from_omnirota" />
			<param name="msg_to_str_sub" value="/fmData/nmea_to_omnirota" />   
			<param name="msg_to_str_pub" value="/fmData/omnirota_tx" />           
			<param name="use_nmea_checksum" type="boolean" value="true" />
		</node>
		<node pkg="serial_string" name="serial_string_node" type="serial_string_node" output="screen" >
			<param name="publisher_topic" value="/fmData/omnirota_rx" />	 
			<param name="subscriber_topic" value="/fmData/omnirota_tx" />
			<param name="device" value="/dev/omnirota_nmea" />	
			<param name="baudrate" value="57600" />
			<param name="termination_character" value="10"/>
		</node>
	</group>
</launch>
